Cargo.toml

```
@@ -12,8 +12,8 @@ serde = { version = "1.0", features = ["derive"] }
 serde_json = "1.0"
 snafu = "0.8"
 stackable-operator = { git = "https://github.com/stackabletech/operator-rs", tag = "stackable-operator-0.86.0" }
-sysinfo = { version = "0.33", features = ["serde"] }
+sysinfo = { version = "0.35.1", features = ["serde"] }
```

nit: For binaries we normally put the minor version in here.
The Cargo.lock takes care of pinning the patch level (this is actually expressed I need 0.35.1 or whatever is the highest patch level)
```suggestion
sysinfo = { version = "0.35", features = ["serde"] }
```

---

Thanks!